pavelib/utils/test/suites/nose_suite.py

```diff
@@ -161,7 +161,7 @@ def __init__(self, *args, **kwargs):
     def cmd(self):
         cmd = (
             "nosetests --id-file={test_ids} {test_id} {test_opts} "
-            "--with-xunit --xunit-file={xunit_report} {extra}"
+            "--with-xunit --xunit-file={xunit_report} {extra} "
```

Comment:
Without the space, this would result in `--with-flaky--verbosity=2` instead of `--with-flaky --verbosity=2` unless you passed an extra space at the end of extra_args.